README.md

```diff
@@ -26,31 +26,19 @@ Shopsys Framework is fully functional e-commerce platform with all basic functio
 * 3-step ordering process
 * Basic CMS
 * Support for several currencies, [languages, and domains](https://docs.shopsys.com/en/latest/introduction/domain-multidomain-multilanguage/)
-* Full friendly URL for main entities
+* Full friendly URL for main entities and full control over SEO aspects of online presence
 * Performance optimization through Elasticsearch, Redis, PostgreSQL
 * Full core upgradability
 * GDPR compliance
 * Preparation for scalability
-* Base for [Back-end API](https://docs.shopsys.com/en/latest/backend-api/)
+* Basic [Back-end API](https://docs.shopsys.com/en/latest/backend-api/)
```

Comment:
I think that Base is more appropriate as there are not many supported functions

Reply:
In my humble opinion, in this context isn't phrase "Base for" the most suitable, that's why I proposed to change it. Let's make it democratic and wait for third opinion?

Reply:
After short slack conversation, we voted for "Basic..."